ykrt/build.rs

```diff
@@ -1,6 +1,59 @@
+use regex::Regex;
 use std::env;
+use std::os::unix::fs::symlink;
+use std::path::PathBuf;
+use std::process::Command;
+
+include!("../build_aux.rs");
+
+const YKSHIM_SO: &str = "libykshim.so";
 
 fn main() {
+    let ykrt_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let mut internal_dir = ykrt_dir.clone();
+    internal_dir.push("..");
+    internal_dir.push("internal_ws");
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let cargo = env::var("CARGO").unwrap();
+    let rust_flags = env::var("RUSTFLAGS").unwrap_or_else(|_| String::new());
+    let tracing_kind = find_tracing_kind(&rust_flags);
+
+    // Build the internal workspace.
+    env::set_current_dir(&internal_dir).unwrap();
+    let status = Command::new(cargo)
+        .arg("build")
+        .arg("--release")
+        .env_remove("RUSTFLAGS")
+        .env(
```

Comment:
This is redundant.

Reply:
Just the `env_remove` is redundant, right?

Reply:
Correct